File: Assets/Scripts/PigOrc.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class PigOrc : MonoBehaviour {

	void Start () {
		GetComponent<Animation>().Play("Idle");
	}

	void OnTriggerEnter (Collider other) {
		if(other.gameObject.tag == "Player"){
			Debug.Log("hmmmp");
			GetComponent<Animation>().Play("BackFlip");
		};
	}

}

File: Assets/Scripts/PigOrc.js
#pragma strict


function Start () {
	GetComponent.<Animation>().Play("Idle");
}

function OnTriggerEnter (other : Collider) {
    if(other.gameObject.tag == "Player"){
    	print('hmmmp');
    	GetComponent.<Animation>().Play("BackFlip");
    };
}
